docs/release.md

```diff
@@ -39,29 +39,59 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This is the same release cycle as Kubernetes, but KubeVirt will trail Kubernetes releases
```

Comment:
"release cycle as Kubernetes" can we link this to https://kubernetes.io/releases/release/